data/presets/highway/busway.json

```diff
@@ -0,0 +1,32 @@
+{
+    "icon": "temaki-bus",
+    "fields": [
+        "name",
+        "operator",
+        "oneway",
+        "structure",
+        "covered"
+    ],
+    "moreFields": [
+        "trolley_wire"
+    ],
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "highway": "busway"
+    },
+    "addTags": {
+        "highway": "busway",
+        "access": "no",
+        "bus": "designated",
+        "emergency": "yes"
+    },
+    "terms": [
+        "busway",
```

Comment:
It would be slightly better to put these as 'aliases'. See [this](https://github.com/ideditor/schema-builder/issues/3).